### PR TITLE
Strip ACF key from response when not an object

### PR DESCRIPTION
**Pull request by chrisgeary92**
Fixes: #2471 

This PR removes `acf: false` from the WP API response.

When a post doesn't have any ACF fields, the WP API is set to `acf: false` which breaks all posts in that post type. Instead, we should just remove `acf: false` as it's not required.

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 4e771a42e0a83821a5d6376bd13e72303dceb6f6

https://app.netlify.com/sites/using-glamor/deploys/59e4fc970b79b77f22960842

File: packages/gatsby-source-wordpress/src/normalize.js [hunk 1]
```diff
@@ -57,6 +57,20 @@ function getValidKey({ key, verbose = false }) {
 
 exports.getValidKey = getValidKey
 
+// Remove the ACF key from the response when it's not an object
+const normalizeACF = entities =>
+  entities.map(e => {
+    Object.keys(e)
```
Comment (KyleAMathews) [inline packages/gatsby-source-wordpress/src/normalize.js]: No need to iterate over keys

**Comment by KyleAMathews** [comment]
👍 

Will wait for tests to pass then merge/release. Thanks!

**Comment by chrisgeary92** [comment]
Awesome! Sorry it took a while. Thanks for the tips!

**Comment by jlengstorf** [comment]
Hiya @crgeary! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**